appengine-java21/bigtable/build.gradle

```diff
@@ -31,8 +31,8 @@ apply plugin: 'com.google.cloud.tools.appengine'
 group = 'com.example.google.cloud.bigtable'
 version = '0.1-SNAPSHOT'
 
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = 17
```

Comment:
should these be 21 instead? it feels a little confusing to me how we would recommend folks to configure these

Reply:
The good news with Java source code is languae level at a previous version works with the latest one...
If we want to keep samples easily moveable between runtimes, it is better to keep the smallest version.
Also Java 21 kokoro is not correct setup. In general, the kokoro tests for the entire repo might still be a global issue.
So I prefer to keep it this way, and it will work with java21 as well.